support-frontend/assets/components/subscriptionCheckouts/billingPeriodSelector.jsx

```diff
@@ -29,6 +29,7 @@ type PropTypes = {|
   billingCountry: IsoCountry,
   selected: BillingPeriod,
   onChange: (BillingPeriod) => Action,
+  orderIsAGift?: boolean | null,
```

Comment:
Use `Option<boolean>` here for consistency

Reply:
Thanks, shall do